src/redux/explorer.ts

```diff
@@ -637,20 +542,13 @@ export const explorerInit = () => async (
   // if there is another socket unsubscribe first
   if (addressSocket || assetsSocket) {
     dispatch(explorerUnsubscribe());
-    dispatch(disableFallbackIfNeeded());
   }
 
-  // Fallback to the testnet data provider
-  // if we're not on mainnnet
   const provider = await getProviderForNetwork(network);
   const providerUrl = provider?.connection?.url;
   checkForTheMerge(provider, network);
```

Comment:
unrelated but should we delete this?

Reply:
yes I have it in my notes to follow up on this